lib/cache/access_list.go

```diff
@@ -250,11 +251,8 @@ func (c *Cache) ListAccessListMembers(ctx context.Context, accessListName string
 		return out, next, trace.Wrap(err)
 	}
 
-	start := accessListName
+	start := cmp.Or(pageToken, accessListName)
 	end := sortcache.NextKey(accessListName + "/")
-	if pageToken != "" {
-		start += "/" + pageToken
-	}
```

Comment:
i just wanted to double check, was this a bug too? b/c the changes don't look equivalent